src/overlays/actors/ovl_En_Ru/z_en_ru.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_en_ru.c
  * Overlay: ovl_En_Ru
- * Description: Ruto
+ * Description: (OoT's Adult) Ruto (unused)
```

Comment:
Shouldn't this be `Adult Ruto` instead?

Reply:
That's what's already there, but the parentheses make it a little unclear. I'll clean it up.